PartyPlaylistsTests/UnitTests/SongDataStoreUnitTests.cs

```diff
@@ -14,28 +15,28 @@ namespace PartyPlaylistsTests.UnitTests
     public class SongDataStoreUnitTests
     {
         private PlaylistContext _playlistContext;
-
+        private SongDataStore songDataStore;
```

Comment:
Add underscore to datastore

```suggestion
        private SongDataStore _songDataStore;
```